spec/unit/daemon/pushy/delivery_spec.rb

```diff
@@ -61,10 +61,12 @@
       it_behaves_like 'process notification'
     end
 
-    shared_examples 'retry delivery' do |response_code:|
-      let(:response_code) { response_code }
+    shared_examples 'retry delivery' do |options|
```

Comment:
In Ruby 3.0, this way of destructuring arguments is no longer supported.

See the following link for more details:
https://bugs.ruby-lang.org/issues/14183#note-114

Reply:
Oh, that's an interesting discussion. I haven't seen it before. Thanks for the link!